go.mod

```diff
@@ -65,9 +65,9 @@ require (
 	google.golang.org/api v0.138.0
 	gotest.tools v2.2.0+incompatible
 	helm.sh/helm/v3 v3.12.3
-	k8s.io/api v0.27.4
-	k8s.io/apimachinery v0.27.4
-	k8s.io/client-go v0.27.4
+	k8s.io/api v0.27.7
```

Comment:
Remainder should be done in a separate PR, as this one is just about the storage changes.